packages/template-chakra-storefront/src/components/product-view-modal/bundle.test.js

```diff
@@ -40,40 +40,46 @@ MockComponent.propTypes = {
 }
 
 beforeEach(() => {
-    global.server.use(
-        rest.get('*/products/:productId', (req, res, ctx) => {
-            return res(ctx.delay(0), ctx.status(200), ctx.json(mockProductBundle))
-        }),
-        rest.get('*/products', (req, res, ctx) => {
-            const swingTankBlackMediumVariantId = '701643473915M'
-            const swingTankBlackLargeVariantId = '701643473908M'
-            if (req.url.toString().includes(swingTankBlackMediumVariantId)) {
-                mockProductBundleWithVariants.data[1].inventory = {
-                    ...mockProductBundleWithVariants.data[1].inventory,
-                    stockLevel: 0
-                }
-            } else if (req.url.toString().includes(swingTankBlackLargeVariantId)) {
-                mockProductBundleWithVariants.data[1].inventory = {
-                    ...mockProductBundleWithVariants.data[1].inventory,
-                    stockLevel: 1
+    prependHandlersToServer([
```

Comment:
using prependHandlersToServer to reduce boilderPlate code for global mock server